src/models/Product.js

```diff
@@ -7,6 +7,7 @@ const productSchema = new mongoose.Schema({
     imageUrl: String,
     quantity: Number,
     rating: Number,
+    cloudinary_id: String,
```

Comment:
Why are we storing `cloudinary_id`, if it is really required, let be consistent here and use camel case here write `cloudinaryId` instead of `cloudinary_id`